README.md

```diff
@@ -1 +1,60 @@
-# hello-rails-back-end
\ No newline at end of file
+# hello-rails-back-end
+
+# Description
+
+This is a new kind of 'Hello World!' app. For this exercise I'm going to create a React front-end with a Rails back-end each in a separate repo and connect them to display a random message.
+
+## Link to front-end repo
+[click here](https://github.com/devMahmoud/hello-react-front-end)
+
+## Built With
+* Ruby on Rails
+* PostgreSQL
+
+## Getting Started
+To get a local copy on this project on your machine follow these steps
+- Open your terminal
+- Run this command:
+`git clone https://github.com/devMahmoud/hello-rails-back-end.git`
```

Comment:
[OPTION] Please add in the front end README the link to your back-end repo because it is mandatory